src/agents/agent.py

```diff
@@ -104,50 +87,35 @@ def run_agent_with_interactive(self) -> None:
 
             user_message = input()
             if user_message == "exit":
-                break
+                return
```

Comment:
```suggestion
                break
```
ここは`bread`のままでよいと思います．
while文を抜けることを明示するためです．

Reply:
おそらく対応するコミットは[tori-suggestion](https://github.com/torippy01/llm_poc/pull/6/commits/bceda4bf2880de0c8b5b013259121a1e1da8954e)だと思います．
指摘に対応するcommitは、該当するフィード等にコメントすると親切ですね！
またcommit名ですが、tori-suggestionではどの指摘を指すcommitか伝わりにくいです．
英語でなんて書くかぱっとは浮かばないですが、`breakに戻す`等書いてもらえると助かります＾＾

Reply:
わかりました！